vector/src/main/java/im/vector/app/features/home/room/list/RoomSummaryListController.kt

```diff
@@ -17,18 +17,26 @@
 package im.vector.app.features.home.room.list
 
 import im.vector.app.features.home.RoomListDisplayMode
+import im.vector.app.features.settings.FontScalePreferences
 import org.matrix.android.sdk.api.session.room.model.RoomSummary
 
 class RoomSummaryListController(
         private val roomSummaryItemFactory: RoomSummaryItemFactory,
-        private val displayMode: RoomListDisplayMode
+        private val displayMode: RoomListDisplayMode,
+        fontScalePreferences: FontScalePreferences
 ) : CollapsableTypedEpoxyController<List<RoomSummary>>() {
 
     var listener: RoomListListener? = null
+    private val shouldUseSingleLine: Boolean
+
+    init {
+        val fontScale = fontScalePreferences.getResolvedFontScaleValue()
+        shouldUseSingleLine = fontScale.scale > FontScalePreferences.SCALE_LARGE
```

Comment:
A tiny detail, but I would have used `>=`
```suggestion
        shouldUseSingleLine = fontScale.scale >= FontScalePreferences.SCALE_LARGER
```

Reply:
I don't feel ok checking equality of two floating-point numbers. We don't do any calculations here, so they should be the same, but I still concerned. 🤔

Reply:
I see your point, OK